modules/cluster-tagging.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="cluster-tagging_{context}"]
+= Tagging an existing cluster as owned
+
+A cluster uses the value of the `kubernetes.io/cluster/<clusterid>,Value=(owned|shared)` tag to determine which resources belong to the AWS cluster. These resources include:
+
+* All hosts.
```

Comment:
Both `owned` and `shared` are acceptable.

Note, for IPI, almost all resources were added `kubernetes.io/cluster/<clusterid>:owned` tag by installer when you creating cluster.

Reply:
So, in the BZ, there was this request: https://bugzilla.redhat.com/show_bug.cgi?id=1872580#c35

That's a follow up to this note from Comment 18:

"- NOTE: Make sure you add the cluster tag to the instance kubernetes.io/cluster/<cluster-id>=owned"

So, I guess my question would be: should it stay as owned in this context, and is that the full cluster getting tagged or or just the RHEL Worker node?

Reply:
@EricPonvelle I don't think `owned` is mandatory, since both `owned` and `shared` work well for IPI and UPI, and I think it is good to let user to choose `owned` or `shared` according to their own requirements